src/tasks/OneDriveLargeFileUploadTask.ts

```diff
@@ -39,6 +39,29 @@ export class OneDriveLargeFileUploadTask extends LargeFileUploadTask {
 	 */
 	private static DEFAULT_UPLOAD_PATH: string = "/";
 
+	/**
+	 * @private
+	 * @static
+	 * Constructs the create session url for Onedrive
+	 * @param {string} fileName - The name of the file
+	 * @param {path} [path = OneDriveLargeFileUploadTask.DEFAULT_UPLOAD_PATH] - The path for the upload
+	 * @returns The constructed create session url
+	 */
+	private static constructCreateSessionUrl(fileName: string, path: string = OneDriveLargeFileUploadTask.DEFAULT_UPLOAD_PATH): string {
+		fileName = fileName.trim();
+		path = path.trim();
+		if (path === "") {
+			path = "/";
+		}
+		if (path[0] !== "/") {
+			path = `/${path}`;
+		}
+		if (path[path.length - 1] !== "/") {
+			path = `${path}/`;
+		}
+		return encodeURI(`/me/drive/root:${path}${fileName}:/createUploadSession`);
```

Comment:
There are many entitysets that can be targeted for uploading a large item. Per docs, at least /drives. /groups, /sites, /users in addition to the /me singleton. We should support all of them. Additionally, we should support not just uploading to the drive root, but to any arbitrary folder specified by itemid.

Reply:
Yes, this is one of the ask from the user and we are helping to achieve the same using large file upload base class. This is what this PR is all about, previously creating upload session with largeFileUpload base class was not possible, this PR adds that.